safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -362,6 +362,14 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 14062206, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 14062208, "1.3.0"),
     ],
+    EthereumNetwork.KAVA_EVM: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2116303, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2116307, "1.3.0"),
+    ],
+    EthereumNetwork.KAVA_EVM_TESTNET: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2241523, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2241523, "1.3.0"),
```

Comment:
In this block only have one transaction to create the proxyFactory. Then I couldn't confirm that this contracts were deployed at this block. 
https://explorer.testnet.kava.io/block/2241523/transactions#txs